docs/disco.sh

```diff
@@ -228,4 +232,12 @@ ketrew-ui () {
     echo https://$EXTERNAL_IP/gui?token=$TOKEN
 }
 
+publish-to-bucket () {
+    local results_directory=$1
+    local bucket=$2
+    local path=$(random-string)
```

Comment:
this makes the command not "restartable"
every `publish` makes a new directory, is this really what we want?

it'd be nice to be able to rerun it when seomthing fails, of if we rerun the same pipeline with more options for example.